bin/elixirc.bat

```diff
@@ -1,15 +1,15 @@
-@if defined ELIXIR_CLI_ECHO (@echo on) else  (@echo off)
+@if defined ELIXIR_CLI_ECHO (@echo on) else (@echo off)
 setlocal
 set argc=0
 for %%A in (%*) do (
-    if /I "%%A"=="--help" goto documentation
-    if /I "%%A"=="-h"     goto documentation
-    if /I "%%A"=="/h"     goto documentation
-    if    "%%A"=="/?"     goto documentation
-    set /A argc+=1
+  if /I "%%A"=="--help" goto :documentation
+  if /I "%%A"=="-h"     goto :documentation
+  if /I "%%A"=="/h"     goto :documentation
+  if    "%%A"=="/?"     goto :documentation
+  set /A argc+=1
 )
-if %argc%==0 goto documentation
-goto run
+if %argc%==0 goto :documentation
+goto :run
```

Comment:
According to the docs, `goto :run` is undocumented while `goto run` is the official mechanism. Maybe we should remove the colons? The only officially supported one with colon is `:eof`: http://ss64.com/nt/goto.html

What do you think?